kanidmd/src/lib/audit.rs

```diff
@@ -35,20 +36,27 @@ macro_rules! audit_log {
  * })
  */
 
-macro_rules! audit_segment {
-    ($au:expr, $fun:expr) => {{
+macro_rules! lperf_segment {
```

Comment:
It makes *so* much sense in production. So many times we get users who say "my server is slow" and by the time they enable the logs, they can't reproduce. Then you get others who are so terrified to enable logs, that they won't even if it would help solve the case.

By always forcing the logs on like this it means we have to make them fast, and it means that admins always have the data there when they say "my server is slow".

Reply:
Cool, when we have performance tests we could see better. If it is free is really cool. I dunno it must be mandatory of optional in other case. But I like your approach.

Reply:
I guess it's my years of working on 389-ds and with customer support that has pushed me in this direction :)